content/intro-curriculum/H_Visualize.md

```diff
@@ -50,7 +50,7 @@ qtemp_gg <- ggplot(data=intro_df, aes(x=Flow_Inst, y=Wtemp_Inst))
 qtemp_gg
 ```
 
-<img src='../static/ggplot2/ggplot_examp-1.png'/ alt='/Blank ggplot2 plot'/>
+<img src='../static/ggplot2/ggplot_examp-1.png'/ title='/Blank ggplot2 plot'/>
```

Comment:
So here's an example where there are R generated figures, and all I really did was update that header chunk.